packages/@aws-cdk/aws-cognito/package.json

```diff
@@ -73,9 +73,11 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@aws-cdk/assert": "0.0.0",
+    "@types/punycode": "^2.1.0",
     "cdk-build-tools": "0.0.0",
     "cdk-integ-tools": "0.0.0",
     "cfn2ts": "0.0.0",
+    "eslint-plugin-node": "^11.1.0",
```

Comment:
Can we use this rule - https://eslint.org/docs/rules/no-restricted-imports - instead, which seems to be available with standard eslint?